ee/tabby-webserver/src/service/license.rs

```diff
@@ -48,6 +49,51 @@ pub fn validate_license(token: &str) -> Result<LicenseInfo, jwt::errors::ErrorKi
     Ok(data?.claims)
 }
 
+fn jwt_timestamp_to_utc(f: f64) -> Result<DateTime<Utc>> {
+    let secs = f as i64;
+    Ok(NaiveDateTime::from_timestamp_millis(secs * 1000)
+        .ok_or_else(|| anyhow!("Timestamp is corrupt"))?
+        .and_utc())
+}
+
+#[async_trait]
+impl LicenseService for DbConn {
```

Comment:
As discussed previously, shall we use a LicensenServiceImpl(DbConn) to avoid potential method refactoring in future?